src/metabase/public_settings.clj

```diff
@@ -816,6 +816,48 @@ See [fonts](../configuring-metabase/fonts.md).")
                     ;; frontend should set this value to `true` after the modal has been shown once
                     v))))
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; v50 Permissions Tutorial settings
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```

Comment:
I think these should actually go into a different namespace. (Maybe `metabase.api.permissions`?) They're not public settings since they're admin-only, and I think they'd make more sense near the other perms-related code.

Sorry if I mislead with an earlier suggestion — I was just referring to moving the `instance-creation` setting which _is_ public.

Reply:
Makes sense, I'll move it there